Makefile

```diff
@@ -22,12 +22,8 @@ LOCALCOMPOSEFILE := ./docker-compose.local.yml
 
 # create a docker-compose binary either as a link or a script
 $(COMPOSEBIN):
-ifneq ($(shell command -v docker-compose 2>/dev/null),)
-	ln -sf $(shell command -v docker-compose 2>/dev/null) $@
-else
 	curl -fsSL "https://github.com/docker/compose/releases/download/1.29.2/run.sh" -o $@
 	chmod +x $@
-endif
```

Comment:
Did you mean to commit this? I haven't tested myself but I assume most docker-compose versions will work?

Reply:
Yes, it was intentional, check out the commit message.